packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -108,6 +112,10 @@ export class GadgetConnection {
     this.websocketsEndpoint = this.websocketsEndpoint.replace(/^http/, "ws");
     this.environment = options.environment ?? "Development";
     this.requestPolicy = options.requestPolicy ?? "cache-and-network";
+    this.rateLimitExchange = rateLimitExchange({
+      enabled: false,
```

Comment:
Disabling this exchange by default so that we can "revert" this if anything goes wrong. We can override this option (and any others) via the area config.

This means that the rate limit will never be enabled in the browser, but I don't think we're concerned about browser clients atm.